lib/order.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'menu'
+
+class Order
+  attr_reader :take_order, :menu
+
+  def initialize(menu = Menu.new)
+    @menu = menu
+    @take_order = []
+  end
+
+  def customer_order(item, amount)
```

Comment:
```suggestion
  def customer_order(item, amount = 1)
```
Setting a default would mean the customer doesn't need a quantity